mantidproject/mslice
# Refactor plot window

Refactor Plot window to base plot window, 

**To test:**
Load workspace2d in mslice and verify that the plot window works by selecting the workspace and clicking display

Fixes #25
